installers/go-agent/release/agent.sh

```diff
@@ -119,13 +118,6 @@ else
   PID_FILE="$AGENT_WORK_DIR/go-agent.pid"
 fi
 
-if [ "$VNC" == "Y" ]; then
```

Comment:
Thinking out aloud, should we warn if this variable is set, perhaps with some like to a documentation about possible fallbacks?

Reply:
Really? I thought a note in the release notes should be fine. Also, is this even a change that would require us to add a deprecation note given that only people who upgrade the bootstrapper would face the problem? I don't mind adding the warning though. What fallbacks? We should ask people to start the vncserver on their own, right?

Reply:
> What fallbacks? We should ask people to start the vncserver on their own, right?

That's what I meant as "fallback", not automated, but something users can use to replace this functionality.

Reply:
I'm not suggesting that we do this, I'm just thinking if there are things we can possibly do to make it easy for users.